src/screens/OtherProfile/elements/BioAndChat.js

```diff
@@ -166,15 +166,15 @@ const styles = StyleSheet.create({
     marginBottom: 10
   },
   bioText: {
-    color: COLORS.white,
+    color: COLORS.white2,
     fontSize: 14,
-    fontWeight: '600',
+    fontFamily: fonts.inter[600],
     lineHeight: 22
   },
   seeMore: {
     fontFamily: fonts.inter[500],
     fontSize: 14,
-    color: COLORS.black
+    color: COLORS.white2
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in text color from `COLORS.white` and `COLORS.black` to `COLORS.white2` may affect readability. Also, replacing fontWeight with fontFamily might cause issues if the specified font family does not exist or is not loaded correctly. Make sure these changes are tested thoroughly across different devices and screen sizes.
```diff
- color: COLORS.white,
+ color: COLORS.white2,
- fontWeight: '600',
+ fontFamily: fonts.inter[600],
- color: COLORS.black
+ color: COLORS.white2
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->